diagnostic.md

```diff
@@ -7,40 +7,49 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+<!-- git@github.com:lcurran/git-basics-diagnostic.git -->
```

Comment:
Correct. But the command is `git clone git@github.com:lcurran/git-basics-diagnostic.git`